lib/notify/notification.rb

```diff
@@ -83,6 +84,19 @@ def send_changed_password_email(email_address:, first_name:)
     Rails.logger.error(e.message)
   end
 
+  def send_admin_changed_user_password_email(email_address:, first_name:, url:)
+    mail_client.send_email(
+      email_address: email_address,
+      template_id: ADMIN_RESET_USER_PASSWORD_TEMPLATE,
+      personalisation: {
+        first_name: first_name,
+        url: "[https://www.admin.verify.service.gov.uk/reset-user-password](#{url})",
```

Comment:
the domain should be taken from the APP_URL env variable, so it's correct in staging

Reply:
Is that not whats happening. By calling the method `#{url}` it returns `"http://#{Rails.configuration.app_url}"`?

Reply:
Does this also matter within the tests? or can i manually define it to the test url? for example `http://www.test.com/reset-user-password` or should it be `http://#{Rails.configuration.app_url}`

Reply:
in test.rb is already defined as `config.app_url = 'www.test.com'` .. so your test should expec that... 

don't you have a name clash? one of your arguments is called `url` and also a further down there's an `url` method... you probablu want here something like:
`url: "[{url}/reset-user-password](#{reset_url})",`
i.e. rename the argument and use the `url` for the hostname, and the passed in path for the rest of the url.

Reply:
Thats exactly what ive already named it to :)

Reply:
ah nice! I'll wait for the final PR then :D ..